packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -276,7 +277,17 @@ class Question {
 	 * @returns {string}
 	 */
 	formatAnswerForSummary(answer) {
-		return this.altText ?? answer ?? 'Not started';
+		let answerValue = this.altText ?? answer ?? 'Not started';
+		let formattedAnswer = '';
+		if (answer != undefined && answer.uploadedFiles) {
+			for (const item in answer.uploadedFiles) {
+				formattedAnswer = formattedAnswer + answer.uploadedFiles[item].originalFileName + '</br>';
```

Comment:
Ideally this would be in the multifile file upload question class rather than the base question.